docs/04-tutorial.md

```diff
@@ -127,6 +127,34 @@ rollup --config rollup.config.dev.js
 rollup --config rollup.config.prod.js
 ```
 
+### Using Rollup locally
+
+When working with multiple environments or in a team, it's often smart to add Rollup as a _local_ dependency. This way you don't have to remember (or remind team members) to install Rollup globally and you can make sure everyone is using the same version.
+
+To install Rollup locally you can simply do:
+
+```console
+npm i -D rollup
+// OR yarn -D add rollup
+```
+
+Now you can run it inside the root folder of your project like so:
+
+```console
+npx rollup --config
+// OR yarn rollup --config
+```
+
+In this case it's usually a good idea to also add a script to your `package.json`, so the whole team uses the correct build command:
+
+```json
+{
+  "scripts": {
+    "build": "npx rollup --config"
```

Comment:
This version doesn't need to use `npx`, it can simply be `"build": "rollup --config"`